src/org/omegat/core/team2/impl/SVNAuthenticationManager.java

```diff
@@ -68,17 +73,17 @@ public class SVNAuthenticationManager implements ISVNAuthenticationManager {
 
     private static final Logger LOGGER = Logger.getLogger(SVNAuthenticationManager.class.getName());
 
-    private final String repoUrl;
+    private final RepositoryDefinition repoDef;
     private final String predefinedUser;
     private final String predefinedPass;
-    //private final ProjectTeamSettings teamSettings;
+    private final ProjectTeamSettings teamSettings;
 
-    public SVNAuthenticationManager(String repoUrl, String predefinedUser, String predefinedPass,
-            ProjectTeamSettings teamSettings) {
-        this.repoUrl = repoUrl;
+    public SVNAuthenticationManager(RepositoryDefinition repoDef, String predefinedUser,
```

Comment:
A change of constructor arguments, it might break compatibility. It may be better to keep a constructor with old signature.